src/helpers/pivot/spreadsheet_pivot/spreadsheet_pivot.ts

```diff
@@ -188,7 +188,8 @@ export class SpreadsheetPivot implements Pivot<SpreadsheetPivotRuntimeDefinition
     if (!finalCell) {
       return { value: "" };
     }
-    if (finalCell.value === null) {
+    // Value can be null but stringified (e.g. an empty date, as for now every date is stringified)
```

Comment:
Null dates are not stringified anymore with the last change right ? Or are they stringified somewhere else ?

Reply:
They are stringified by the caller